### PR TITLE
feat(containers): update limits from recent api changes

### PR DESCRIPTION
**Pull request by thomas-tacquet**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: serverless/containers/reference-content/containers-limitations.mdx [hunk 1]
```diff
@@ -16,27 +16,27 @@ categories:
...
+| Containers                                        | Max number     | 1000*             | Organization                |
+| Total container memory\**                         | Max size       | 600 GiB           | Organization                |
+| Image size compressed                             | Max size       | 250 MiB           | Container                   |
+| Image size uncompressed                           | Max size       | 1 GiB             | Container                   |
+| Temporary disk size                               | Max size       | 1024 MiB          | Container                   |
+| Invocation rate                                   | Max number     | 1000 per second   | Container                   |
+| Concurrency                                       | Max            | 80                | Container Instance          |
+| Max Scale (simultaneous Container instances)      | Max            | 50                | Container                   |
+| CPU                                               | Min/max        | 100 to 6000 mvCPU | Container                   |
```
Comment (Bemilie) [inline serverless/containers/reference-content/containers-limitations.mdx:30]:
```suggestion
| CPU                                               | Min/max        | 70 to 6000 mvCPU | Container                   |
```

File: serverless/containers/reference-content/containers-limitations.mdx [hunk 2]
```diff
@@ -16,27 +16,27 @@ categories:
...
+| Memory                                            | Min/Max        | 128 to 12228 MiB  | Container                   |
```
Comment (Bemilie) [inline serverless/containers/reference-content/containers-limitations.mdx:31]:
```suggestion
| Memory                                            | Min/Max        | 128 to 12228 MB  | Container                   |
```

File: serverless/containers/reference-content/containers-limitations.mdx [hunk 3]
```diff
@@ -71,10 +71,10 @@ In order to ensure the proper functioning of the product, we restrict the use of
...
 When container CPU and/or memory are not provided by the client, these default values are automatically used:
 
-| Resources | Default value |
-|-----------|---------------|
-| CPU       | 140 mvCPU     |
-| Memory    | 256 MiB       |
+| Resources | Default value  |
+|-----------|----------------|
+| CPU       | 1000 mvCPU     |
+| Memory    | 2048 MiB       |
```
Comment (Bemilie) [inline serverless/containers/reference-content/containers-limitations.mdx:77]:
```suggestion
| Memory    | 2048 MB       |
```

**Review by Bemilie — COMMENTED** [review]
This is not related to the recent changes, but all memory and disk size values/limits are in MB/GB, not in MiB/GiB. This can be checked in [this protobuf file](https://gitlab.infra.online.net/protobuf/protobuf/-/blob/master/scaleway/containers/v1beta1/api.yml?ref_type=heads)

**Review by nerda-codes — APPROVED** [review]

**Review by Bemilie — APPROVED** [review]

**Pull request merged**